### PR TITLE
Feature/sdl 0262 new vehicle data seat occupancy

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Implements https://github.com/smartdevicelink/sdl_core/issues/3152

This PR is **[ready]** for review.

### Risk
This PR makes **[minor]** API changes.

### Testing Plan
ATF test scripts from https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2490

### Summary
Add new vehicle data SeatOccupancy
Use rpc spec from https://github.com/smartdevicelink/rpc_spec/pull/298

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@atiwari9 please review this PR

File: .gitmodules
Comment (atiwari9) [inline .gitmodules:6]: Temporary commit? Should not be part of actual donation..

**Review by atiwari9 — APPROVED** [review]
Approved contingent on test scripts Pass test result for: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2490

File: .gitmodules
Comment (LitvinenkoIra) [reply .gitmodules:6]: @atiwari9 Yes, you are right, this is a temporary commit with the link to the rpc_spec that contains MOBILE API changes. We need it just for testing and it will be deleted before the merge

**Comment by LitvinenkoIra** [comment]
@JackLivio This PR is ready for Livio review. Thank you!

File: src/appMain/sdl_preloaded_pt.json
Comment (jacobkeeler) [inline src/appMain/sdl_preloaded_pt.json:637]: Would it be better to to include this in `DrivingCharacteristics-3`? I don't have a strong opinion on the matter, but `beltStatus` is included in that group, and that has a fairly similar function to this field.

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: src/appMain/sdl_preloaded_pt.json
Comment (LitvinenkoIra) [reply src/appMain/sdl_preloaded_pt.json]: Done in 1f41ade8d1b740ad9f386ccc2820ba0d4d034726

File: src/appMain/sdl_preloaded_pt.json
Comment (LitvinenkoIra) [reply src/appMain/sdl_preloaded_pt.json]: Done in 1f41ade8d1b740ad9f386ccc2820ba0d4d034726

File: .gitmodules
Comment (LitvinenkoIra) [reply .gitmodules]: @jacobkeeler Done in d420676

File: src/appMain/sdl_preloaded_pt.json
Comment (LitvinenkoIra) [reply src/appMain/sdl_preloaded_pt.json]: @jacobkeeler Done in 12201aa

File: src/components/interfaces/HMI_API.xml
Comment (jacobkeeler) [inline src/components/interfaces/HMI_API.xml:1812]:
```suggestion
  <param name="seatLocation" type="Common.SeatLocation" mandatory="true"/>
```

**Review by jacobkeeler — APPROVED** [review]

**Comment by jacobkeeler** [comment]
@LitvinenkoIra please fix conflicts (should be as simple as updating rpc_spec submodule to latest develop)

**Pull request merged**
